Jenkinsfile

```diff
@@ -84,7 +84,30 @@ elifePipeline {
             }
         }
 
-        stage 'Browser Tests', {
+        stage 'Integration Tests', {
+            def actions = [
```

Comment:
sure these two can run in parallel? I remember making browser tests use a different Postgres logical database to avoid clashes with the rest of the tests

Reply:
(I am not sure if the separate databases are used anymore because I see a single `elife-xpub_setupdb`)

Reply:
I haven't seen any clashes between the two, so I'm not sure if it's an issue - I'll add a note so if it does become a problem, people will know what to change

Reply:
I'm pretty sure that both of these parallel instances are running against the same instance of the application so I'd assume they're both pointing to the same database.

Reply:
There really shouldn't be any clashes, anything that would clash (e.g. manuscript title) is generated uniquely per test